include/mgp.hpp

```diff
@@ -2590,6 +2653,26 @@ inline bool Node::operator==(const Node &other) const { return util::NodesEqual(
 
 inline bool Node::operator!=(const Node &other) const { return !(*this == other); }
 
+inline const std::string Node::ToString() const {
+  std::string labels{""};
+  for (auto label : Labels()) {
+    labels += ":" + std::string(label) + " ";
+  }
+  std::string properties = "";
+  std::map<std::string, Value> properties_map{Properties()};
+  const size_t map_size = properties_map.size();
+  size_t i = 0;
+  for (auto item : properties_map) {
+    if (i == map_size - 1) {
+      properties += std::string(item.first) + "=" + item.second.ToString();
+      break;
+    }
+    properties += std::string(item.first) + "=" + item.second.ToString() + ", ";
```

Comment:
For properties, we usually return `"key : "value"` instead of `"key" = "value"`. I think we should change that here too